packages/checkout-ui-extensions/src/components/shared.ts

```diff
@@ -16,7 +16,7 @@ export interface Autocomplete {
   field: AutocompleteField;
 }
 
-export type AutocompleteGroup = 'shipping' | 'billing';
+export type AutocompleteGroup = 'shipping' | 'billing' | 'location';
```

Comment:
This type change would be an enhancement, not a breaking change as it is additive in nature.